docs/source/fitting/fitfunctions/ChudleyElliotSQE.rst

```diff
@@ -0,0 +1,48 @@
+.. _func-Chudley-ElliotSQE:
+
+================
+ChudleyElliotSQE
+================
+
+.. index:: ChudleyElliotSQE
+
+Description
+-----------
+
+This fitting function models the dynamic structure factor
+for a particle undergoing jump diffusion using the Chudley-Elliot Jump diffusion model [1]_.
+
+.. math::
+
+   S(Q,E) = Height \cdot \frac{1}{\pi} \frac{\Gamma}{\Gamma^2+(E-Centre)^2}
+
+   \Gamma = \frac{\hbar}{\tau} \cdot \left(1 - \frac{sin(Ql)}{Ql}\right)
+
+where:
+
+-  :math:`Height` - Intensity scaling, a fit parameter
+-  :math:`Centre` - Centre of peak, a fit parameter
+-  :math:`Tau` - Residence time, a fit parameter
+-  :math:`l` - Jump lengths, a fit parameter
+-  :math:`Q` - Momentum transfer, an attribute (non-fitting)
+
```

Comment:
It would be helpful if you also mentioned what Gamma and E represent. Are these half width at half maximum (HWHM) and the energy transfer, respectively? Is Gamma a function of Q?

Reply:
Gamma is HWHM, E is energy tranasfer. The Gamma variation with Q is in the equation!

Reply:
Right, Gamma and E are used in the equation, so they should be probably explained in the documentation. Please disregard the comment about Gamma = Gamma(Q), as I compared this file to a wrong documentation file (FickDiffusion instead of FickDiffusionSQE) and thought both notations should be better consistent with each other... Also, I am not sure if the name "Height" describes well the corresponding parameter, as it does not seem to depict the peak's height. But I leave it to you to decide if that's important or not

Reply:
I've had a look some other peak functions. Most use Height and a few Amplitude. I think we'll keep Height.